## librtrad-location.dpatch by Bernd Zeimetz <bernd@bzed.de>
##
## DP: put librtrad in the right directory while building

@DPATCH@
--- a/src/common/Rmakefile
+++ b/src/common/Rmakefile
@@ -37,27 +37,24 @@ MGFOBJ = mgf_parser.o mgf_object.o mgf_x
 
 OGLOBJ = rglfile.o rglmat.o rgldomat.o rglsurf.o rglinst.o rglsrc.o
 
-LIBFILES = tmesh.cal
+LIBFILES = 
 
-librtrad.a:	$(RTOBJ) $(PICOBJ) $(UTLOBJ) $(STDOBJ) $(SYSOBJ)
-	rm -f librtrad.a
-	ar rc librtrad.a $(RTOBJ) $(PICOBJ) $(UTLOBJ) $(STDOBJ) $(SYSOBJ)
-	-ranlib librtrad.a
-
-libmgf.a:	$(MGFOBJ)
-	rm -f libmgf.a
-	ar rc libmgf.a $(MGFOBJ)
-	-ranlib libmgf.a
+../lib/librtrad.a:	$(RTOBJ) $(PICOBJ) $(UTLOBJ) $(STDOBJ) $(SYSOBJ)
+	rm -f $@
+	ar rc $@ $(RTOBJ) $(PICOBJ) $(UTLOBJ) $(STDOBJ) $(SYSOBJ)
+	-ranlib $@
+
+../lib/libmgf.a:	$(MGFOBJ)
+	rm -f $@
+	ar rc $@ $(MGFOBJ)
+	-ranlib $@
 
 install:	all
-	mv -f librtrad.a libmgf.a ../lib
-	cd $(LIBDIR) && rm -f $(LIBFILES)
-	cp -f $(LIBFILES) $(LIBDIR)
 
-all:	librtrad.a libmgf.a $(SPECIAL)
+all:	../lib/librtrad.a ../lib/libmgf.a $(SPECIAL)
 
 clean:
-	rm -f *.o librtrad.a libmgf.a
+	rm -f *.o ../lib/librtrad.a ../lib/libmgf.a
 
 ogl:	$(OGLOBJ)
 	rm -f librgl.a
